package.json

```diff
@@ -2,26 +2,29 @@
   "name": "chromatography",
   "version": "2.0.6",
   "description": "Tools for storing, search and analize GC/MS spectra",
-  "main": "src/index.js",
+  "main": "lib/index.js",
+  "module": "src/index.js",
+  "files": [
+    "lib",
+    "src"
+  ],
   "scripts": {
     "build": "cheminfo build --no-uglify --root Chromatography",
```

Comment:
@targos should I keep this build task as that or anything has changed for the modules support?